src/Word/Service/SeparatorToSeparatorFactory.php

```diff
@@ -23,7 +24,7 @@ public function __construct($creationOptions = null)
             return;
         }
 
-        if ($creationOptions instanceof Traversable) {
+        if ($creationOptions instanceof \Traversable) {
```

Comment:
Isn't this double? I think it's either `use Traversable;` or `instanceof \Traversable`. Using `\Traversable` makes the import obsolete. Importing is preferred though.